README.md

```diff
@@ -629,43 +629,28 @@ scripts.yaml:
       required: true
   sequence:
     - variables:
-        cfg_ids: "{{ state_attr('sensor.enphase_schedules', 'cfg') or [] }}"
-        dtg_ids: "{{ state_attr('sensor.enphase_schedules', 'dtg') or [] }}"
-        rbd_ids: "{{ state_attr('sensor.enphase_schedules', 'rbd') or [] }}"
-    - choose:
-        - conditions: "{{ schedule_type in ['cfg', 'all'] }}"
-          sequence:
-            - repeat:
-                for_each: "{{ cfg_ids }}"
```

Comment:
Issue is here: If `schedule_type` is `all`, then only `cfg_ids` get deleted. No further choose blocks are evaluated.